pkg/printers/internalversion/printers.go

```diff
@@ -1884,3 +1894,55 @@ func printControllerRevisionList(list *apps.ControllerRevisionList, options prin
 	}
 	return rows, nil
 }
+
+func printResourceQuota(resourceQuota *api.ResourceQuota, options printers.PrintOptions) ([]metav1beta1.TableRow, error) {
+	row := metav1beta1.TableRow{
+		Object: runtime.RawExtension{Object: resourceQuota},
+	}
+
+	resources := make([]api.ResourceName, 0, len(resourceQuota.Status.Hard))
+	for resource := range resourceQuota.Status.Hard {
+		resources = append(resources, resource)
+	}
+	sort.Sort(SortableResourceNames(resources))
+
+	var requestColumn string
+	var limitColumn string
+	for i := range resources {
+		resource := resources[i]
+		usedQuantity := resourceQuota.Status.Used[resource]
+		hardQuantity := resourceQuota.Status.Hard[resource]
+
+		if strings.HasPrefix(resource.String(), "request") {
```

Comment:
Based on the docs for ResourceQuotas, it looks like we can consider resources "requests" as long as they do not have the "limits." prefix. Also, does not make sense to repeat ourselves here when populating the "REQUEST" and "LIMIT" columns. You could modify your look to be a bit simpler, for example:

```go
requestColumn := bytes.NewBuffer([]byte{})
limitColumn := bytes.NewBuffer([]byte{})
for i := range resources {
	w := requestColumn
	resource := resources[i]
	usedQuantity := resourceQuota.Status.Used[resource]
	hardQuantity := resourceQuota.Status.Hard[resource]

	// use limitColumn writer if we find a resource name prefixed with "limits"
	if pieces := strings.Split(resource.String(), "."); len(pieces) > 1 && pieces[0] == "limits" {
		w = limitColumn
	}

	fmt.Fprintf(w, "%s: %s/%s, ", resource, usedQuantity.String(), hardQuantity.String())
}

age := translateTimestampSince(resourceQuota.CreationTimestamp)
row.Cells = append(row.Cells, resourceQuota.Name, age)
//if options.Wide {
	row.Cells = append(row.Cells, strings.TrimSuffix(requestColumn.String(), ", "), strings.TrimSuffix(limitColumn.String(), ", "))
//}
```

Reply:
Link to docs for ResourceQuota: https://kubernetes.io/docs/concepts/policy/resource-quotas/#compute-resource-quota

Reply:
Additionally, it could make sense to display all resources, not just `cpu` and `memory`. The example above does this by not special-casing on a resource's name

Reply:
Yeah, its better to show all resources and this loop is definitely more efficient. I've made the necessary updates.